{
  "uid" : "e2ac2468ae3f5463",
  "name" : "Verify amount and visibility of sections with content on the page",
  "fullName" : "tests.start_unauthorized_page_test.TestStartUnauthorizedPageStructure#test_su_01_02_verify_sections_structure_and_visibility",
  "historyId" : "813003ea9cdf1ba0de6a80a14f886de6",
  "time" : {
    "start" : 1728781332392,
    "stop" : 1728781332392,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5631bac0b02a <unknown>\n#1 0x5631ba8f15e0 <unknown>\n#2 0x5631ba8e97eb <unknown>\n#3 0x5631ba8d9a09 <unknown>\n#4 0x5631ba8db744 <unknown>\n#5 0x5631ba8d9cbd <unknown>\n#6 0x5631ba8d952f <unknown>\n#7 0x5631ba8d9422 <unknown>\n#8 0x5631ba8d7493 <unknown>\n#9 0x5631ba8d7ada <unknown>\n#10 0x5631ba8f3f41 <unknown>\n#11 0x5631ba9858c5 <unknown>\n#12 0x5631ba965b22 <unknown>\n#13 0x5631ba984d7d <unknown>\n#14 0x5631ba9658c3 <unknown>\n#15 0x5631ba9336b3 <unknown>\n#16 0x5631ba93468e <unknown>\n#17 0x5631babd5a2b <unknown>\n#18 0x5631babd99b1 <unknown>\n#19 0x5631babc2225 <unknown>\n#20 0x5631babda532 <unknown>\n#21 0x5631baba738f <unknown>\n#22 0x5631babf9f28 <unknown>\n#23 0x5631babfa0f3 <unknown>\n#24 0x5631bac09e7c <unknown>\n#25 0x7f7d69694ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"8f429901b3e376d44e0031da81284a2d\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c0fc9eb0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5631babfa0f3 \\\\u003Cunknown>\\\\n#24 0x5631bac09e7c \\\\u003Cunknown>\\\\n#25 0x7f7d69694ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5631bac0b02a <unknown>\nE       #1 0x5631ba8f15e0 <unknown>\nE       #2 0x5631ba8e97eb <unknown>\nE       #3 0x5631ba8d9a09 <unknown>\nE       #4 0x5631ba8db744 <unknown>\nE       #5 0x5631ba8d9cbd <unknown>\nE       #6 0x5631ba8d952f <unknown>\nE       #7 0x5631ba8d9422 <unknown>\nE       #8 0x5631ba8d7493 <unknown>\nE       #9 0x5631ba8d7ada <unknown>\nE       #10 0x5631ba8f3f41 <unknown>\nE       #11 0x5631ba9858c5 <unknown>\nE       #12 0x5631ba965b22 <unknown>\nE       #13 0x5631ba984d7d <unknown>\nE       #14 0x5631ba9658c3 <unknown>\nE       #15 0x5631ba9336b3 <unknown>\nE       #16 0x5631ba93468e <unknown>\nE       #17 0x5631babd5a2b <unknown>\nE       #18 0x5631babd99b1 <unknown>\nE       #19 0x5631babc2225 <unknown>\nE       #20 0x5631babda532 <unknown>\nE       #21 0x5631baba738f <unknown>\nE       #22 0x5631babf9f28 <unknown>\nE       #23 0x5631babfa0f3 <unknown>\nE       #24 0x5631bac09e7c <unknown>\nE       #25 0x7f7d69694ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781333172,
      "stop" : 1728781333761,
      "duration" : 589
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5631bac0b02a <unknown>\n#1 0x5631ba8f15e0 <unknown>\n#2 0x5631ba8e97eb <unknown>\n#3 0x5631ba8d9a09 <unknown>\n#4 0x5631ba8db744 <unknown>\n#5 0x5631ba8d9cbd <unknown>\n#6 0x5631ba8d952f <unknown>\n#7 0x5631ba8d9422 <unknown>\n#8 0x5631ba8d7493 <unknown>\n#9 0x5631ba8d7ada <unknown>\n#10 0x5631ba8f3f41 <unknown>\n#11 0x5631ba9858c5 <unknown>\n#12 0x5631ba965b22 <unknown>\n#13 0x5631ba984d7d <unknown>\n#14 0x5631ba9658c3 <unknown>\n#15 0x5631ba9336b3 <unknown>\n#16 0x5631ba93468e <unknown>\n#17 0x5631babd5a2b <unknown>\n#18 0x5631babd99b1 <unknown>\n#19 0x5631babc2225 <unknown>\n#20 0x5631babda532 <unknown>\n#21 0x5631baba738f <unknown>\n#22 0x5631babf9f28 <unknown>\n#23 0x5631babfa0f3 <unknown>\n#24 0x5631bac09e7c <unknown>\n#25 0x7f7d69694ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781333173,
        "stop" : 1728781333761,
        "duration" : 588
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5631bac0b02a <unknown>\n#1 0x5631ba8f15e0 <unknown>\n#2 0x5631ba8e97eb <unknown>\n#3 0x5631ba8d9a09 <unknown>\n#4 0x5631ba8db744 <unknown>\n#5 0x5631ba8d9cbd <unknown>\n#6 0x5631ba8d952f <unknown>\n#7 0x5631ba8d9422 <unknown>\n#8 0x5631ba8d7493 <unknown>\n#9 0x5631ba8d7ada <unknown>\n#10 0x5631ba8f3f41 <unknown>\n#11 0x5631ba9858c5 <unknown>\n#12 0x5631ba965b22 <unknown>\n#13 0x5631ba984d7d <unknown>\n#14 0x5631ba9658c3 <unknown>\n#15 0x5631ba9336b3 <unknown>\n#16 0x5631ba93468e <unknown>\n#17 0x5631babd5a2b <unknown>\n#18 0x5631babd99b1 <unknown>\n#19 0x5631babc2225 <unknown>\n#20 0x5631babda532 <unknown>\n#21 0x5631baba738f <unknown>\n#22 0x5631babf9f28 <unknown>\n#23 0x5631babfa0f3 <unknown>\n#24 0x5631bac09e7c <unknown>\n#25 0x7f7d69694ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"8f429901b3e376d44e0031da81284a2d\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781332392,
      "stop" : 1728781333172,
      "duration" : 780
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781333794,
      "stop" : 1728781333856,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Start Unauthorized Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "start_unauthorized_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestStartUnauthorizedPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.start_unauthorized_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 6,
        "skipped" : 0,
        "passed" : 493,
        "unknown" : 0,
        "total" : 499
      },
      "items" : [ {
        "uid" : "5f346774c1d8ed1c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/5f346774c1d8ed1c",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x557825c9602a <unknown>\n#1 0x55782597c5e0 <unknown>\n#2 0x5578259747eb <unknown>\n#3 0x557825964a09 <unknown>\n#4 0x557825966744 <unknown>\n#5 0x557825964cbd <unknown>\n#6 0x55782596452f <unknown>\n#7 0x557825964422 <unknown>\n#8 0x557825962493 <unknown>\n#9 0x557825962ada <unknown>\n#10 0x55782597ef41 <unknown>\n#11 0x557825a108c5 <unknown>\n#12 0x5578259f0b22 <unknown>\n#13 0x557825a0fd7d <unknown>\n#14 0x5578259f08c3 <unknown>\n#15 0x5578259be6b3 <unknown>\n#16 0x5578259bf68e <unknown>\n#17 0x557825c60a2b <unknown>\n#18 0x557825c649b1 <unknown>\n#19 0x557825c4d225 <unknown>\n#20 0x557825c65532 <unknown>\n#21 0x557825c3238f <unknown>\n#22 0x557825c84f28 <unknown>\n#23 0x557825c850f3 <unknown>\n#24 0x557825c94e7c <unknown>\n#25 0x7f4709894ac3 <unknown>",
        "time" : {
          "start" : 1728694658248,
          "stop" : 1728694658248,
          "duration" : 0
        }
      }, {
        "uid" : "afcc184753e670a0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/afcc184753e670a0",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55a756a9d02a <unknown>\n#1 0x55a7567835e0 <unknown>\n#2 0x55a75677b7eb <unknown>\n#3 0x55a75676ba09 <unknown>\n#4 0x55a75676d744 <unknown>\n#5 0x55a75676bcbd <unknown>\n#6 0x55a75676b52f <unknown>\n#7 0x55a75676b422 <unknown>\n#8 0x55a756769493 <unknown>\n#9 0x55a756769ada <unknown>\n#10 0x55a756785f41 <unknown>\n#11 0x55a7568178c5 <unknown>\n#12 0x55a7567f7b22 <unknown>\n#13 0x55a756816d7d <unknown>\n#14 0x55a7567f78c3 <unknown>\n#15 0x55a7567c56b3 <unknown>\n#16 0x55a7567c668e <unknown>\n#17 0x55a756a67a2b <unknown>\n#18 0x55a756a6b9b1 <unknown>\n#19 0x55a756a54225 <unknown>\n#20 0x55a756a6c532 <unknown>\n#21 0x55a756a3938f <unknown>\n#22 0x55a756a8bf28 <unknown>\n#23 0x55a756a8c0f3 <unknown>\n#24 0x55a756a9be7c <unknown>\n#25 0x7f08fd694ac3 <unknown>",
        "time" : {
          "start" : 1728608261951,
          "stop" : 1728608261951,
          "duration" : 0
        }
      }, {
        "uid" : "c7ef9f3efafcb310",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/c7ef9f3efafcb310",
        "status" : "passed",
        "time" : {
          "start" : 1728557402493,
          "stop" : 1728557402524,
          "duration" : 31
        }
      }, {
        "uid" : "3869c784d33a504f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/3869c784d33a504f",
        "status" : "passed",
        "time" : {
          "start" : 1728554268287,
          "stop" : 1728554268319,
          "duration" : 32
        }
      }, {
        "uid" : "768682acfd4dedb4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/768682acfd4dedb4",
        "status" : "passed",
        "time" : {
          "start" : 1728522277458,
          "stop" : 1728522277491,
          "duration" : 33
        }
      }, {
        "uid" : "f9daaa0318863d27",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/f9daaa0318863d27",
        "status" : "passed",
        "time" : {
          "start" : 1728470667802,
          "stop" : 1728470667835,
          "duration" : 33
        }
      }, {
        "uid" : "4deda4b9929ff6e3",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/4deda4b9929ff6e3",
        "status" : "passed",
        "time" : {
          "start" : 1728468096291,
          "stop" : 1728468096327,
          "duration" : 36
        }
      }, {
        "uid" : "5711875dffa66dbc",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/5711875dffa66dbc",
        "status" : "passed",
        "time" : {
          "start" : 1728435904520,
          "stop" : 1728435904556,
          "duration" : 36
        }
      }, {
        "uid" : "86a827e5897a0329",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/86a827e5897a0329",
        "status" : "passed",
        "time" : {
          "start" : 1728378112182,
          "stop" : 1728378112214,
          "duration" : 32
        }
      }, {
        "uid" : "a8f333f196d907ea",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/a8f333f196d907ea",
        "status" : "passed",
        "time" : {
          "start" : 1728375320811,
          "stop" : 1728375320856,
          "duration" : 45
        }
      }, {
        "uid" : "2413b049c452ffb2",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/2413b049c452ffb2",
        "status" : "passed",
        "time" : {
          "start" : 1728375202445,
          "stop" : 1728375202480,
          "duration" : 35
        }
      }, {
        "uid" : "1c2f5673299662fa",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/1c2f5673299662fa",
        "status" : "passed",
        "time" : {
          "start" : 1728349291063,
          "stop" : 1728349291098,
          "duration" : 35
        }
      }, {
        "uid" : "788f2ef91658278d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/788f2ef91658278d",
        "status" : "passed",
        "time" : {
          "start" : 1728293040960,
          "stop" : 1728293040994,
          "duration" : 34
        }
      }, {
        "uid" : "cfdb035e07cd8457",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/cfdb035e07cd8457",
        "status" : "passed",
        "time" : {
          "start" : 1728290090973,
          "stop" : 1728290091006,
          "duration" : 33
        }
      }, {
        "uid" : "2dab9b28b691816c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/2dab9b28b691816c",
        "status" : "passed",
        "time" : {
          "start" : 1728263738091,
          "stop" : 1728263738131,
          "duration" : 40
        }
      }, {
        "uid" : "9c103a5f3d3025f6",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/9c103a5f3d3025f6",
        "status" : "passed",
        "time" : {
          "start" : 1728226913633,
          "stop" : 1728226913667,
          "duration" : 34
        }
      }, {
        "uid" : "6f71ec07174e327b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/6f71ec07174e327b",
        "status" : "passed",
        "time" : {
          "start" : 1728222005691,
          "stop" : 1728222005724,
          "duration" : 33
        }
      }, {
        "uid" : "62f594ac5a2e9aa0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/62f594ac5a2e9aa0",
        "status" : "passed",
        "time" : {
          "start" : 1728177101374,
          "stop" : 1728177101409,
          "duration" : 35
        }
      }, {
        "uid" : "88107b8715449384",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/88107b8715449384",
        "status" : "passed",
        "time" : {
          "start" : 1728117733747,
          "stop" : 1728117733792,
          "duration" : 45
        }
      }, {
        "uid" : "c961c2806ba0eaae",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/c961c2806ba0eaae",
        "status" : "passed",
        "time" : {
          "start" : 1728109111639,
          "stop" : 1728109111671,
          "duration" : 32
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "e2ac2468ae3f5463.json",
  "parameterValues" : [ ]
}